src/status_im/subs/communities.cljs

```diff
@@ -73,6 +73,13 @@
  (fn [community _]
    (js-keys (:members community))))
 
+(re-frame/reg-sub
+ :communities/community-members-count
+ (fn [[_ community-id]]
+   (re-frame/subscribe [:communities/community-members community-id]))
+ (fn [members _]
+   (count members)))
```

Comment:
This was causing a re-render when visiting a community, now it's cached so we are saving one re-render.

As far as I tested, the screen doesn't re-render by external factors not related.